prombench/manifests/cluster-infra/grafana_deployment.yaml

```
@@ -72,12 +66,9 @@ spec:
       - name: grafana-dashboard-provision
         configMap:
           name: grafana-dashboard-provision
-      - name: grafana-dashboard-prombench
+      - name: grafana-dashboard-dashboards
         configMap:
-          name: grafana-dashboard-prombench
-      - name: grafana-dashboard-node-metrics
-        configMap:
-          name: grafana-dashboard-node-metrics
+          name: grafana-dashboard-dashboards
```

why not just 
```suggestion
          name: grafana-dashboards
```

the same at the other place

---

okay adding the change